### ISSUE TITLE
Bug: Unable to export scene to glb2 due to json writer failure

### ISSUE THREAD

**Issue by ghost** [assimp/assimp#4727]
Hi,

**Summary**
I am using assimp to import an obj+mtl mesh and export to gltf2 in the form of glb2. I encountered a failure when writing json. This zip contains the obj and mtl file [wrist3.zip](https://github.com/assimp/assimp/files/9586310/wrist3.zip). 
The version I see failures in v5.2.4 and v5.2.5. After bisecting previous commits, I found this failure started at [this commit](https://github.com/assimp/assimp/commit/aef4ecada533c40d41ae4e885eaa39bfe74601cb). 

I would like to know how this mesh file is invalid or see if there is a bug in the glb2 export process.

**To Reproduce**
1. Import the obj to an assimp scene [wrist3.zip](https://github.com/assimp/assimp/files/9586310/wrist3.zip). 
2. Export the assimp scene to with `exporter.ExportToBlob(scene, "glb2")

**Expected behavior**
Expect the export to fail with error message 
```
in void rapidjson::Writer<rapidjson::GenericStringBuffer<rapidjson::UTF8<>>, rapidjson::UTF8<>, rapidjson::UTF8<>, rapidjson::CrtAllocator, 0>::Prefix(Type) [OutputStream = rapidjson::GenericStringBuffer<rapidjson::UTF8<>>, SourceEncoding = rapidjson::UTF8<>, TargetEncoding = rapidjson::UTF8<>, StackAllocator = rapidjson::CrtAllocator, writeFlags = 0]: type == kStringType
```
Based on my call stack, this error happens at this [line](https://github.com/assimp/assimp/blob/v5.2.5/code/AssetLib/glTF2/glTF2AssetWriter.inl#L821)

**Platform (please complete the following information):**
 - OS: Linux

**Comment by tellypresence** [comment]
Closing due to author account deleted

**Comment by kimkulling** [comment]
This is right; the author no longer exists. But the issue is still relevant. Please do not close issues without checking if this is still there.

**Comment by tellypresence** [comment]
Sorry, my bad.

I wanted to ask if it might be possible to please add a new label named something like "Still current" or "Still active" for old bugs that should not be closed (would be very useful for triage to filter out bugs as they've been checked).  Also, was looking at google's [`filament`](https://github.com/google/filament) and saw they have labels "high priority" and "low priority" -- seems like similar labels would be very useful for assimp if you agree

**Comment by kimkulling** [comment]
This is a good idea. And do not worry. Thanks for the great work you are currently doing. It is a little bit challenging for me to keep uptodate.

**Comment by tellypresence** [comment]
Thank you -- while you're adding labels, please also check these lists for any you agree might be useful
- #6097
- #6113

**Comment by kimkulling** [comment]
I will do it. I was on vacation last week.

**Comment by kimkulling** [comment]
I did some investigations on this issue. It seems, that the exported representation for the scene as a JSON file gets corrupted somehow.  There is a loop going over every value in the JSON tree. The writer's stuff is implemented using a visitor pattern. Somehow the accept-method fails at **assimp\contrib\rapidjson\include\rapidjson\document.h** in line 1959. I dug into the code and for me, the Accept-method will return true. Unfortunately, the debugger says it will return false, and the whole export failed.

**Comment by kimkulling** [comment]
The problem occurs when assimp tries to generate the json structure for the materials.

**Comment by tellypresence** [comment]
glTF export issues tracked via epic #6147
